src/main/java/it/gov/pagopa/payhub/activities/service/ingestionflow/assessmentsregistry/AssessmentsRegistryProcessingService.java

```diff
@@ -0,0 +1,91 @@
+package it.gov.pagopa.payhub.activities.service.ingestionflow.assessmentsregistry;
+
+import com.opencsv.exceptions.CsvException;
+import it.gov.pagopa.payhub.activities.connector.classification.AssessmentsRegistryService;
+import it.gov.pagopa.payhub.activities.dto.ingestion.assessmentsregistry.AssessmentsRegistryErrorDTO;
+import it.gov.pagopa.payhub.activities.dto.ingestion.assessmentsregistry.AssessmentsRegistryIngestionFlowFileDTO;
+import it.gov.pagopa.payhub.activities.dto.ingestion.assessmentsregistry.AssessmentsRegistryIngestionFlowFileResult;
+import it.gov.pagopa.payhub.activities.mapper.ingestionflow.assessmentsregistry.AssessmentsRegistryMapper;
+import it.gov.pagopa.payhub.activities.service.ingestionflow.IngestionFlowProcessingService;
+import it.gov.pagopa.pu.classification.dto.generated.AssessmentsRegistry;
+import it.gov.pagopa.pu.processexecutions.dto.generated.IngestionFlowFile;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Service;
+
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+@Service
+@Lazy
+@Slf4j
+public class AssessmentsRegistryProcessingService extends
+        IngestionFlowProcessingService<AssessmentsRegistryIngestionFlowFileDTO, AssessmentsRegistryIngestionFlowFileResult, AssessmentsRegistryErrorDTO> {
+
+    private final AssessmentsRegistryMapper assessmentsRegistryMapper;
+    private final AssessmentsRegistryService assessmentsRegistryService;
+
+    public AssessmentsRegistryProcessingService(
+            AssessmentsRegistryMapper assessmentsRegistryMapper,
+            AssessmentsRegistryErrorsArchiverService assessmentsRegistryErrorsArchiverService,
+            AssessmentsRegistryService assessmentsRegistryService) {
+        super(assessmentsRegistryErrorsArchiverService);
+        this.assessmentsRegistryMapper = assessmentsRegistryMapper;
+        this.assessmentsRegistryService = assessmentsRegistryService;
+    }
+
+    public AssessmentsRegistryIngestionFlowFileResult processAssessmentsRegistry(
+            Iterator<AssessmentsRegistryIngestionFlowFileDTO> iterator,
+            List<CsvException> readerException,
+            IngestionFlowFile ingestionFlowFile, Path workingDirectory) {
+
+        List<AssessmentsRegistryErrorDTO> errorList = new ArrayList<>();
+
+        AssessmentsRegistryIngestionFlowFileResult ingestionFlowFileResult = new AssessmentsRegistryIngestionFlowFileResult();
+        ingestionFlowFileResult.setFileVersion(ingestionFlowFile.getFileVersion());
```

Comment:
here you should set the file version related to the csv header read... actually there should not be a version, so let's define it as 1.0
```suggestion
        ingestionFlowFileResult.setFileVersion("1.0");
```